core/src/main/java/com/schibsted/spt/data/jslt/impl/BuiltinFunctions.java

```diff
@@ -1303,8 +1305,17 @@ public JsonNode call(JsonNode input, JsonNode[] arguments) {
         if(aURL.getUserInfo() != null && !aURL.getUserInfo().isEmpty())
           objectNode.put("userinfo", aURL.getUserInfo());
         return objectNode;
-      } catch (MalformedURLException | UnsupportedEncodingException e) {
-        throw new JsltException("Can't parse " + urlString, e);
+      } catch (Throwable e) {
```

Comment:
Possible regression as this catch is different. However, we ran into `InvalidArgumentException` when having trailing `%` which with the current code pattern gave no hints on the problem string.

Reply:
`Throwable` is perhaps overdoing it, since it's going to catch `OutOfMemoryError` and similar things. Perhaps just `Exception`?

Reply:
`Throwable` -> `Exception` makes sense 👍  We just rushed the fix out without considering that.

Reply:
Or we can simulate [`NonFatal`](https://github.com/scala/scala/blob/cbe6ade1744bb316a63a07b07d3bf448590ad2cd/src/library/scala/util/control/NonFatal.scala#L41) :)

Reply:
Now is `Exception`